src/components/railBottom/RailBottom.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="rail-container">
+    <div class="rail-left">
+      <MiniPaperControl />
+
+      <ZoomControl
+        :paper-manager="paperManager"
+      />
+    </div>
+
+    <div class="rail-center">
+      <UndoRedoControl />
+
+      <Controls />
+    </div>
+  </div>
+</template>
+
+<script>
+import MiniPaperControl from '@/components/railBottom/miniPaperControl/MiniPaperControl.vue';
+import ZoomControl from '@/components/railBottom/zoomControl/ZoomControl.vue';
+import UndoRedoControl from '@/components/railBottom/undoRedoControl/UndoRedoControl.vue';
+import Controls from '@/components/railBottom/controls/Controls.vue';
+
+export default {
+  components: {
+    MiniPaperControl,
+    ZoomControl,
+    UndoRedoControl,
+    Controls,
+  },
+  props: {
+    paperManager: {},
+  },
```

Comment:
```suggestion
  props: ['paperManager'],
```
since we are not checking for the prop type or anything, just declare it like this